egs2/babel/asr1/README.md

```diff
@@ -0,0 +1,86 @@
+# BABEL RECIPE
+
+## Setting Up Experiments
+To setup an experiment (using one or more languages in training) simply run
+from this directory ...
+
+`./setup_experiment.sh <expname>`
+
+This will copy all the necessary files for running an experiment to the created
+directory ../expname
+
+Possibly the most important of these files is 
+
+`conf/lang.conf`
```

Comment:
I didn't realized this file, but did this recipe write the database path to lang.conf in babel?
Hmm, kaldi uses really not consistent style between tasks...  Unbelievable...

Sorry, we must unify the style as db.sh, so could you change? Like BABEL_CANTONESE, ...

Reply:
![Screen Shot 2020-03-01 at 22 14 58](https://user-images.githubusercontent.com/22814472/75642607-19c52400-5c0a-11ea-9bd9-90a6e21ce11f.png)
Yes, basically it has the dataset path, but in a very rough manner like this...

Reply:
Babel also employ lexicon file for initializing, can I put their paths in `db.sh` as well

Reply:
The shell variable in db.sh is supposed to correspond to the root directory of corpus which downloaded from original source as it is, e.g LDC.
I haven't get babel corpus, so I don't know what level is the root. Isn't the lexicon included at the same data corpus?

I guess as following:

```
# Cantonese
train_data_dir_101=$LDC2016S02/release-current/conversational/training
train_data_list_101=./conf/lists/101-cantonese/train.LimitedLP.list
train_data_dir_101_FLP=$LDC2016S02//release-current/conversational/training
train_data_list_101_FLP=./conf/lists/101-cantonese/train.FullLP.list
dev10h_data_dir_101=$LDC2016S02/release-current/conversational/dev
dev10h_data_list_101=./conf/lists/101-cantonese/dev.list
lexicon_file_101=$LDC2016S02/release-current/conversational/reference_materials/lexicon.sub-train1.txt
lexiconFlags_101="--romanized --oov <unk>"
```

( I just found where unk is inserted, the last line is )

Reply:
Gotcha! I will finish this format issue soon